README.md

```diff
@@ -4,7 +4,9 @@
 
 This Terraform module will create an [Amazon Elastic Container Registry](https://aws.amazon.com/ecr/) private repository for use on the Cloud Platform.
 
-If you're using GitHub as your OIDC provider, this module will automatically create the required variables for authentication in your GitHub repository.
+If you're using GitHub as your OIDC provider, this module will automatically create the required variables for authentication in your GitHub repository. By default these will be created as [repository secrets and variables](https://docs.github.com/en/actions/learn-github-actions/variables#creating-configuration-variables-for-a-repository). Optionally, you can configure the module to create the ECR secrets
```

Comment:
Optionally sounds like additionally. Can we say something like, if you have defined [GitHub Environments](https://docs.github.com/en/actions/learn-github-actions/variables#creating-configuration-variables-for-an-environment), you can configure the module to create the ECR secrets and variables into your own defined Github Environment and not as a Github repository secrets. This can be done by passing the `github_environments` with the value you defined for yourr Github environment.

Reply:
Good point, I've updated it. Hopefully that clarifies the difference?